ftw/simplelayout/locales/de/LC_MESSAGES/ftw.simplelayout.po

```diff
@@ -82,6 +86,26 @@ msgstr "Video"
 msgid "Youtube format: http(s)://youtu.be/VIDEO_ID<br/>Vimeo format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
 msgstr "Youtube Format: http(s)://youtu.be/VIDEO_ID<br/>Vimeo Format: http(s)://vimeo.com/(channels/groups)/VIDEO_ID"
 
+#. Default: "This item can not be deleted because it is currently locked by another user."
+#: ./ftw/simplelayout/browser/ajax/templates/block_delete_confirmation.pt:6
+msgid "alert_deleting_locked_item"
+msgstr "Dieser Eintrag kann nicht gelöscht werden da er gerade von einem anderen Benutzer blockiert ist."
```

Comment:
`blockiert` -> `gesperrt`.
Plone translates it with `gesperrt`, so we should be consistent with Plone translations.